package.json

```diff
@@ -3,12 +3,14 @@
   "version": "2.0.13",
   "description": "Midi parser & player engine for browser or Node. Works well with single or multitrack MIDI files.",
   "main": "build/index.js",
+  "browser": "build/index.browser.js",
   "scripts": {
     "pretest": "npm run build",
     "test": "mocha",
-    "build": "mkdir -p build && rollup -c && npm run docs",
```

Comment:
I removed the `mkdir` call, which causes issues on Windows. Rollup seems to work fine without it.

Reply:
Great!